### PR TITLE
Add lucet-runtime-internals sigstack allocation vuln advisory

### PR DESCRIPTION
**Pull request by jfoote**
Hello,

This PR is for a security advisory in https://github.com/bytecodealliance/lucet, /cc @acfoltzer.

I am not sure about the format used in the `patched_versions` value. If that is incorrect could you please advise?

If you need anything else on this just let me know.

Thanks,
Jon

### CODE REVIEW

File: crates/lucet-runtime-internals/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+An embedding using affected versions of lucet-runtime configured to use
+non-default Wasm globals sizes of more than 4KiB, or compiled in debug mode
+without optimizations, could leak data from the signal handler stack to guest
+programs. This can potentially cause data from the embedding host to leak to
+guest programs or cause corruption of guest program memory.
+
+This flaw was resolved by correcting the sigstack allocation logic.
+"""
+
+patched_versions = ["< 0.5.0, >= 0.4.3", “>= 0.5.1”]
```
Comment (alex) [inline crates/lucet-runtime-internals/RUSTSEC-0000-0000.toml]:
```suggestion
patched_versions = ["< 0.5.0, >= 0.4.3", ">= 0.5.1"]
```

File: crates/lucet-runtime-internals/RUSTSEC-0000-0000.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+url = "https://github.com/bytecodealliance/lucet/pull/401”
```
Comment (alex) [inline crates/lucet-runtime-internals/RUSTSEC-0000-0000.toml]:
```suggestion
url = "https://github.com/bytecodealliance/lucet/pull/401"
```

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2020-0004` in #231

**Comment by jfoote** [comment]
Thanks @tarcieri and @alex !

**Pull request merged**